packages/coinstac-simulator/src/boot-client.js

```diff
@@ -201,7 +193,7 @@ const kickoff = function kickoff() {
 process.on('message', (opts) => {
   if (opts.boot) {
     boot(opts.boot)
-    .then((result) => process.send({ ready: true, result }));
```

Comment:
`result` is now the return value of `PipelineRunnerPool#init`, which contains self-references. This causes `process.send` to crash.